src/ui/transform.js

```diff
@@ -0,0 +1,26 @@
+import moment from 'moment';
+import type { Message } from '../store/types';
+import { unescape } from '../utility/misc';
+
+// TODO don't mix validation and data munging
```

Comment:
Lolol @ the word `munging`. I've never heard that before.

Reply:
I feel like 90% of my job is data munging, so